ldes-server-infra-postgres/postgres-fragmentation-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/postgres/entity/MemberBucketisationEntity.java

```diff
@@ -0,0 +1,56 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.postgres.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Index;
+import jakarta.persistence.Table;
+
+import static be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.postgres.entity.MemberBucketisationEntity.BUCKETISATION;
+
+@Entity
+@Table(name = BUCKETISATION, indexes = {
+        @Index(columnList = "viewName"),
+        @Index(columnList = "fragmentId"),
+        @Index(columnList = "sequenceNr")
+})
+public class MemberBucketisationEntity {
+    public static final String BUCKETISATION = "bucketisation";
+    @Id
+    private String id;
```

Comment:
if not using the String as an identifier, I'd use a Auto increment value as ID and store the bucket name as a separate value. This mainly has impact on peformance when it comes to indexing.